apps/pie-docs/src/_includes/home-page.njk

```diff
@@ -0,0 +1,29 @@
+{% set heroSrc = ['/assets/img/heroes/', heroImageSrc] | join | url %}
+<div class="c-heroContentLayout" data-test-id="site_content">
```

Comment:
not a request, just a question - is there any way to reduce the number of divs? If not, no worries at all 👍

Reply:
I don't think so sadly, I've just checked and they all serve a purpose. removing them makes things go a bit wonky 😄